exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceArtifactId.java

```diff
@@ -24,8 +24,9 @@
 import com.google.auto.value.AutoValue;
 
 /**
- * A service artifact identifier. It consist of the three coordinates that usually identify any
- * Java artifact: groupId, artifactId and version.
+ * A service artifact identifier. It consist of the runtime id in which the service shall be
+ * deployed and the three coordinates that usually identify any Java artifact: groupId, artifactId
```

Comment:
and the service artifact name. The name of Java artifacts usually contains the three coordinates identifying any Java artifact: ...